src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -200,23 +219,22 @@
   </kd-help-section>
 
   <kd-help-section>
-    <md-switch ng-model="ctrl.runAsPrivileged" class="md-primary">
-      Run as privileged
-    </md-switch>
+    <div>
+      <md-checkbox ng-model="ctrl.runAsPrivileged"
+                   class="md-primary"
+                   layout="row">
```

Comment:
Why do you need `layout=row` here? Layouts only apply to child elements, whose this checkbox does not have